config/routes.rb

```diff
@@ -24,7 +24,7 @@
 
   resources :notifications, only: [:index, :update]
 
-  resources :projects, only: [:show] do
+  resources :projects, only: [:index, :show] do
```

Comment:
What's this for?

Reply:
just for Pro parity down here:

https://github.com/dradis/dradis-ce/pull/934/files/b81de648c088ab4b371ade481eae126e9fa70831#diff-e5e534608b42f749939e20a14f9379d20abb07e787c1533fcac6cdfa15a16068R61

Turns out Projects#index was already implemented in CE:

https://github.com/dradis/dradis-ce/blob/main/app/controllers/projects_controller.rb#L11